src/main.rs

```diff
@@ -138,14 +141,27 @@ async fn main() -> anyhow::Result<()> {
         .with(tracing_subscriber::fmt::layer())
         .init();
 
+    let redirect_url = format!("http://{addr}/authorized");
     let app = Router::new()
+        .route("/login", get(auth::login))
+        .route("/logout", get(auth::logout))
+        .route("/authorized", get(auth::authorized))
         .route("/:uuid/", get(uuid_get))
         .route("/:uuid/out", post(uuid_out_post))
         .route("/:uuid/err", post(uuid_err_post))
         .route(
             "/",
-            get(root_get).post(move |mp| root_post(mp, command, timeout, jobs)),
+            get(root_get).post(move |mp| root_post(None, mp, command, timeout, jobs)),
```

Comment:
If claims are hardcoded to `None`, then we do not allow anyone to schedule workloads. These should be extracted from the request instead